From 12772f0e2788069795a622713d206816dbc83480 Mon Sep 17 00:00:00 2001
From: Christian Simon <simon@swine.de>
Date: Thu, 18 Oct 2018 14:48:39 +0100
Subject: [PATCH] Integrate Google G Suite credentials plugin

Easier way of running the plugin without the stateful plugin setup
---
 command/commands.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/command/commands.go b/command/commands.go
index 1efc8f9f1..ca39c7c3f 100644
--- a/command/commands.go
+++ b/command/commands.go
@@ -37,6 +37,7 @@ import (
 	auditSocket "github.com/hashicorp/vault/builtin/audit/socket"
 	auditSyslog "github.com/hashicorp/vault/builtin/audit/syslog"
 
+	credGoogle "github.com/jetstack/vault-plugin-auth-google/google"
 	credAliCloud "github.com/hashicorp/vault-plugin-auth-alicloud"
 	credAzure "github.com/hashicorp/vault-plugin-auth-azure"
 	credCentrify "github.com/hashicorp/vault-plugin-auth-centrify"
@@ -118,6 +119,7 @@ var (
 		"plugin":     plugin.Factory,
 		"radius":     credRadius.Factory,
 		"userpass":   credUserpass.Factory,
+		"google":     credGoogle.Factory,
 	}
 
 	logicalBackends = map[string]logical.Factory{
-- 
2.19.1

